framework/core/js/src/common/utils/string.ts

```diff
@@ -16,7 +16,8 @@ export function truncate(string: string, length: number, start: number = 0): str
 export function slug(string: string): string {
   return string
     .toLowerCase()
-    .replace(/[^a-z0-9]/gi, '-')
+    // Match non-word characters (take UTF8 into consideration) and replace with a dash.
+    .replace(/[^\p{L}\p{N}\p{M}]/giu, '-')
```

Comment:
The UI uses the `slug` util in core, the util is only used in tags, so I wonder why it is in core. In any case if an extension is using this util, it's not necessarily a breaking change, but definitely quite a change.